package.json

```diff
@@ -202,6 +202,9 @@
     },
     "coffee-script": "$coffeescript"
   },
+  "imports": {
+    "#/*": "./jsapp/*"
```

Comment:
Why is this different from what we have in `tsconfig.json`?

Reply:
oh typo :facepalm: thanks!

FYI it does nothing. It's there in case any of scripts would like to import any of source files in future.